<?xml version="1.0" ?>
<sdf version="1.5">
<model name="pacmouse">
  <pose>0 0 0.05 0 0 0</pose>
  <static>false</static>
    <link name="chassis">
      <pose>0 0 0 0 0 0</pose>
      <inertial>
        <mass>0.1</mass>
        <inertia>
          <ixx>0.00001456041666666667</ixx>
          <ixy>0.00000000000000000000</ixy>
          <ixz>0.00000000000000000000</ixz>
          <iyy>0.00005031532653333334</iyy>
          <iyz>0.00000000000000000000</iyz>
          <izz>0.00005445907653333334</izz>
        </inertia>
      </inertial>
      <collision name="chassis_collision">
        <geometry>
          <box>
            <size>0.073572 0.0335 0.025</size>
          </box>
        </geometry>
      </collision>
      <visual name="chassis_visual">
        <geometry>
          <box>
            <size>0.073572 0.0335 0.025</size>
          </box>
        </geometry>
        <material>
          <ambient>0.1 1.0 0.1 1</ambient>
          <diffuse>0.1 0.1 0.1 1</diffuse>
        </material>
      </visual>
    </link>
    <link name="right_front_wheel">
      <pose>0.018382 -0.033498 0.0 1.5707963267948966 0 0</pose>
      <visual name="right_front_wheel_visual">
        <geometry>
          <cylinder>
            <radius>0.01763</radius>
            <length>0.009144</length>
          </cylinder>
        </geometry>
        <material>
          <ambient>0.0 0.0 0.0 1</ambient>
          <diffuse>0.1 0.1 0.1 1</diffuse>
        </material>
      </visual>
      <collision name="right_front_wheel_collision">
        <geometry>
          <cylinder>
            <radius>0.01763</radius>
            <length>0.009144</length>
          </cylinder>
        </geometry>
        <surface>
          <friction>
            <ode>
              <mu>0.8</mu>
              <mu2>0.8</mu2>
            </ode>
          </friction>
        </surface>
      </collision>
      <inertial>
        <mass>0.03</mass>
        <inertia>
          <ixx>0.00001457461780000000</ixx>
          <ixy>0.00000000000000000000</ixy>
          <ixz>0.00000000000000000000</ixz>
          <iyy>0.00001457461780000000</iyy>
          <iyz>0.00000000000000000000</iyz>
          <izz>0.00002775569000000000</izz>
        </inertia>
      </inertial>
    </link>
    <link name="right_rear_wheel">
      <pose>-0.018382 -0.033498 0.0 1.5707963267948966 0 0</pose>
      <visual name="right_rear_wheel_visual">
        <geometry>
          <cylinder>
            <radius>0.01763</radius>
            <length>0.009144</length>
          </cylinder>
        </geometry>
        <material>
          <ambient>0.0 0.0 0.0 1</ambient>
          <diffuse>0.1 0.1 0.1 1</diffuse>
        </material>
      </visual>
      <collision name="right_rear_wheel_collision">
        <geometry>
          <cylinder>
            <radius>0.01763</radius>
            <length>0.009144</length>
          </cylinder>
        </geometry>
        <surface>
          <friction>
            <ode>
              <mu>0.8</mu>
              <mu2>0.8</mu2>
            </ode>
          </friction>
        </surface>
      </collision>
      <inertial>
        <mass>0.03</mass>
        <inertia>
          <ixx>0.00001457461780000000</ixx>
          <ixy>0.00000000000000000000</ixy>
          <ixz>0.00000000000000000000</ixz>
          <iyy>0.00001457461780000000</iyy>
          <iyz>0.00000000000000000000</iyz>
          <izz>0.00002775569000000000</izz>
        </inertia>
      </inertial>
    </link>
    <link name="left_front_wheel">
      <pose>0.018382 0.033498 0.0 -1.5707963267948966 0 0</pose>
      <visual name="left_front_wheel_visual">
        <geometry>
          <cylinder>
            <radius>0.01763</radius>
            <length>0.009144</length>
          </cylinder>
        </geometry>
        <material>
          <ambient>0.0 0.0 0.0 1</ambient>
          <diffuse>0.1 0.1 0.1 1</diffuse>
        </material>
      </visual>
      <collision name="left_front_wheel_collision">
        <geometry>
          <cylinder>
            <radius>0.01763</radius>
            <length>0.009144</length>
          </cylinder>
        </geometry>
        <surface>
          <friction>
            <ode>
              <mu>0.8</mu>
              <mu2>0.8</mu2>
            </ode>
          </friction>
        </surface>
      </collision>
      <inertial>
        <mass>0.03</mass>
        <inertia>
          <ixx>0.00001457461780000000</ixx>
          <ixy>0.00000000000000000000</ixy>
          <ixz>0.00000000000000000000</ixz>
          <iyy>0.00001457461780000000</iyy>
          <iyz>0.00000000000000000000</iyz>
          <izz>0.00002775569000000000</izz>
        </inertia>
      </inertial>
    </link>
    <link name="left_rear_wheel">
      <pose>-0.018382 0.033498 0.0 -1.5707963267948966 0 0</pose>
      <visual name="left_rear_wheel_visual">
        <geometry>
          <cylinder>
            <radius>0.01763</radius>
            <length>0.009144</length>
          </cylinder>
        </geometry>
        <material>
          <ambient>0.0 0.0 0.0 1</ambient>
          <diffuse>0.1 0.1 0.1 1</diffuse>
        </material>
      </visual>
      <collision name="left_rear_wheel_collision">
        <geometry>
          <cylinder>
            <radius>0.01763</radius>
            <length>0.009144</length>
          </cylinder>
        </geometry>
        <surface>
          <friction>
            <ode>
              <mu>0.8</mu>
              <mu2>0.8</mu2>
            </ode>
          </friction>
        </surface>
      </collision>
      <inertial>
        <mass>0.03</mass>
        <inertia>
          <ixx>0.00001457461780000000</ixx>
          <ixy>0.00000000000000000000</ixy>
          <ixz>0.00000000000000000000</ixz>
          <iyy>0.00001457461780000000</iyy>
          <iyz>0.00000000000000000000</iyz>
          <izz>0.00002775569000000000</izz>
        </inertia>
      </inertial>
    </link>
    <joint name="right_front_wheel_joint" type="revolute">
      <pose>0 0 0 0 0 0</pose>
      <parent>chassis</parent>
      <child>right_front_wheel</child>
      <axis><xyz>0 0 1</xyz></axis>
    </joint>
    <joint name="right_rear_wheel_joint" type="revolute">
      <pose>0 0 0 0 0 0</pose>
      <parent>chassis</parent>
      <child>right_rear_wheel</child>
      <axis><xyz>0 0 1</xyz></axis>
    </joint>
    <joint name="left_front_wheel_joint" type="revolute">
      <pose>0 0 0 0 0 0</pose>
      <parent>chassis</parent>
      <child>left_front_wheel</child>
      <axis><xyz>0 0 1</xyz></axis>
    </joint>
    <joint name="left_rear_wheel_joint" type="revolute">
      <pose>0 0 0 0 0 0</pose>
      <parent>chassis</parent>
      <child>left_rear_wheel</child>
      <axis><xyz>0 0 1</xyz></axis>
    </joint>
</model>
</sdf>
